internal/scm/gitea/gitea.go

```diff
@@ -305,6 +307,85 @@ func (g *Gitea) getLabelsFromStrings(ctx context.Context, repo repository, label
 	return ret, nil
 }
 
+func (g *Gitea) setReviewers(ctx context.Context, repo scm.Repository, newPR scm.NewPullRequest, createdPR *gitea.PullRequest) error {
+	r := repo.(repository)
```

Comment:
Use `repository` directly in the argument instead since this is not a public method.